packages/cli/src/cmds/stats/stats.ts

```diff
@@ -0,0 +1,258 @@
+            // A return's parent_id matches the call's id
+            let functionKey = 'thread_' + event.thread_id + '_id_' + event.parent_id;
+            if (functionKey in functionExecutionTimes[fileName].functions &&
+                functionExecutionTimes[fileName].functions[functionKey].callNoReturnFound) {
+              functionExecutionTimes[fileName].functions[functionKey].callNoReturnFound = false;
+              if ('elapsed_instrumentation' in event) {
+                functionExecutionTimes[fileName].functions[functionKey].elapsedInstrumentationTimeTotal += event.elapsed_instrumentation;
+              }
+              functionExecutionTimes[fileName].functions[functionKey].numberOfCalls += 1;
+            } // else something went wrong: found return with no call
+          }
+        });
+      });
+      verbose(v);
+
+      // now that all functions were collected, index them by function
+      // name, not by <thread_id,parent_id>
+      let totalFunctionExecutionTimes = {}
+      for (var fileName in functionExecutionTimes) {
+        for (var functionKey in functionExecutionTimes[fileName].functions) {
+          let name = functionExecutionTimes[fileName].functions[functionKey].name;
+          if (!(name in totalFunctionExecutionTimes)) {
+            totalFunctionExecutionTimes[name] = {
+              elapsedInstrumentationTimeTotal: 0,
+              numberOfCalls: 0,
+            }
+          }
+          totalFunctionExecutionTimes[name].elapsedInstrumentationTimeTotal += functionExecutionTimes[fileName].functions[functionKey].elapsedInstrumentationTimeTotal;
+          totalFunctionExecutionTimes[name].numberOfCalls += 1;
+        }
+      }
+
+      // convert hash to array
+      let flatFunctionExecutionTimes: {name: string,
+                                       elapsedInstrumentationTimeTotal: number,
+                                       numberOfCalls: number}[] = [];
+      for (var name in totalFunctionExecutionTimes) {
+        flatFunctionExecutionTimes.push({
+          name: name,
+          elapsedInstrumentationTimeTotal: totalFunctionExecutionTimes[name].elapsedInstrumentationTimeTotal,
+          numberOfCalls: totalFunctionExecutionTimes[name].numberOfCalls,
+        });
+      }
+
+
+      return flatFunctionExecutionTimes;
+    }
+
+    async function sortExecutionTimes(functionExecutionTimes) {
+      // sort the array
+      return functionExecutionTimes.sort((a, b) => b.elapsedInstrumentationTimeTotal - a.elapsedInstrumentationTimeTotal);
+    }
+
+    async function showStats() {
+      UI.status = `Computing AppMap stats...`;
+      let appMapDir = directoryToUse;
+      let appMapSizes = await calculateAppMapSizes(appMapDir);
+      let sortedAppMapSizes = await sortAppMapSizes(appMapSizes);
+
+      UI.status = `Displaying biggest appmaps...\n`;
+      let biggestAppMapSizes: {size: number,
+                               name: string}[] = [];
+      let minAppMapSize = 1024*1024/2;
```

Comment:
It would be better to make this a constant named `MINIMUM_APPMAP_SIZE` and put it at the top of the file so it's easier to change / maintain.